### PR TITLE
Removing all packs that are not PANW from marketplace

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5744

## Description
 removing all packs that are not PANW products and that require setting up an instance:
- UrlScan.zip
- FeedTAXII.zip
- Active_Directory_Query.zip
- VirusTotal.zip
- ipinfo.zip

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Review by ShahafBenYakir — CHANGES_REQUESTED** [review]
Before merging this PR need to make sure there are no dependencies that are outside of the core packs list.

File: Tests/Marketplace/core_packs_mpv2_list.json [hunk 1]
```diff
@@ -23,10 +21,7 @@
     "Phishing",
```
Comment (Shellyber) [inline Tests/Marketplace/core_packs_mpv2_list.json:21]: @maimorag We Need to remove from MPv2 Phishing as well.
See the ticket comments

**Review by Shellyber — CHANGES_REQUESTED** [review]

File: Tests/Marketplace/core_packs_mpv2_list.json [hunk 2]
```diff
@@ -53,10 +45,7 @@
...
```
Comment (Shellyber) [inline Tests/Marketplace/core_packs_mpv2_list.json:45]: Also what about the Phishing here?

**Comment by maimorag** [comment]
verified using the UI and also the graph using the following queries:

MATCH (p:Pack {object_id:“**_pack that stays_**”})-[:DEPENDS_ON {mandatorily: true}]-(n)
WHERE n.deprecated IS NULL AND n.object_id IN [“UrlScan”, “FeedTAXII”, “Active_Directory_Query”, “VirusTotal”, “ipinfo”]
RETURN n.object_id

MATCH (p:Pack {object_id:“**_trimmed pack_**”})<-[:DEPENDS_ON {mandatorily: true, is_test: false, marketplaces: ‘marketplacev2’}]-(n) where n.deprecated is null
RETURN n.object_id

**Review by Shellyber — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
@maimorag what is **trimmed pack**? 
Let's also do: 
```
MATCH (p:Pack {object_id:“pack that stays”})-[:DEPENDS_ON {mandatorily: true}]->(n)
WHERE n.deprecated IS NULL
RETURN n.object_id
```
It is without the list of packs you removed.
Go over the resulting list and make sure each item is a core pack.

**Comment by maimorag** [comment]
@ShahafBenYakir trimmed packs are one of the following:[“UrlScan”, “FeedTAXII”, “Active_Directory_Query”, “VirusTotal”, “ipinfo”].

Also, I run the query, and each item is a core pack.

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**